tracerite/style.css

```diff
@@ -135,3 +167,28 @@
   border-collapse: collapse;
   text-align: right;
 }
+
+[data-tooltip] {
```

Comment:
Add to every rule something like `.tracerite .traceback-details` (notebooks define a lot of CSS that we need to override by being more specific).